app/main/helpers/suppliers.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+
+
+def load_countries():
+    helpers_path = os.path.abspath(os.path.dirname(__file__))
+    countryfile = os.path.join(helpers_path, '../../static/location-autocomplete-canonical-list.json')
+    with open(countryfile) as f:
+        return json.load(f)
+
+
+def get_country_name_from_country_code(country_code):
+    if country_code and country_code != 'gb':
```

Comment:
I can't quite articulate why but I want to suggest swapping the order of the logic statements so the special case comes first e.g.
```
if country_code and country_code == 'gb':
elif country_code:
else:
```

I think I like the special case being first and then more normal behaviour continuing afterwards but I'm not too fussed eitherway

Reply:
What you are referring to is called a guard statement I think and I agree that it's a good practice :)

http://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/

Reply:
Yeah, I see what you mean. Something about checking the negative case rather than the positive feels a bit odd.